### DISCUSSION TITLE
CopyQ : ESC key not to minimize app

### DISCUSSION THREAD

**Discussion by Clemens-Ratte-Polle** [hluk/CopyQ#2373] [Q&A]
hi.
how can i stop CopyQ from minimizing to the tray by pressing too much the ESC key?
I cant find this option to unbind this key.
thx

**Comment by hluk** [comment]
You can override the key with the following dummy command ([here is how to add the command to CopyQ](http://copyq.readthedocs.io/en/latest/faq.html#how-to-load-shared-commands-and-share-them)):

```ini
[Command]
Command="
    copyq:"
Icon=\xf15b
InMenu=true
Name=Override ESC
Shortcut=esc
```

**Comment by Clemens-Ratte-Polle** [comment]
ok, thx :)
But what i meant was only not to close the CQ window, the ESC key function to end searches or mark text elements should be unchanged.
thx
